compiler/cpp/src/parser/ast/BooleanExprAST.h

```diff
@@ -0,0 +1,18 @@
+//
+// Created by Marc on 20.05.2021.
+//
+
+#ifndef COMPILER_BOOLEANEXPRAST_H
+#define COMPILER_BOOLEANEXPRAST_H
+
+#include "ValueExprAST.h"
+
+class BooleanExprAST : public ValueExprAST {
+private:
+    bool Value;
```

Comment:
We need a PR whose task it is to change all variables to lower-case 😇

Reply:
Yes. Would consider that as well. The upper-cased attributes are historic remnants